src/rendering/iframe-renderer.tsx

```diff
@@ -1,895 +1,911 @@
+  "www.pcworld.com",
+  "www.unqualified-reservations.org",
+  "evaotaku.com",
+  "forre.st",
+  "mikepower.pressfolios.com",
+  "www.coyneoftherealm.com",
+  "wiki.openttdcoop.org",
+  "quinndunki.com",
+  "learning.mpi-sws.org",
+  "jov.arvojournals.org",
+  "jakewestfall.org",
+  "e2eml.school",
+  "defector.com",
+  "davidepstein.com",
+  "corpgov.law.harvard.edu",
+  "copilot.github.com",
+  "constancecrozier.com",
+  "complearn.org",
+  "compdemocracy.org",
+  "catonmat.net",
+  "calhoun.nps.edu",
+  "bwc.thelab.dc.gov",
+  "blog.thinkst.com",
+  "blog.regehr.org",
+  "blog.cryptographyengineering.com",
+  "b-ok.cc",
+  "archive.seattletimes.com",
+  "architext.design",
+  "andymatuschak.org",
+  "alexanderetz.com",
+  "advertising-effects.chicagobooth.edu",
+  "about.google",
+  "6thfloor.blogs.nytimes.com",
+  "www2.biology.ualberta.ca",
+  "www.wesjones.com",
+  "www.urbandharma.org",
+  "www.tlmc.eu",
+  "www.rfreitas.com",
+  "www.moserware.com",
+  "ew.com",
+  "examples.yourdictionary.com",
+  "fs.blog",
+  "freedomdefined.org",
+  "fibery.io",
+  "fastmoe.ai",
+  "joshmitteldorf.scienceblog.com",
+  "jonmillward.com",
+  "johncwright.livejournal.com",
+  "joa.sh",
+  "jgeekstudies.org",
+  "jdlm.info",
+  "jaypsong.blog",
+  "jax.readthedocs.io",
+  "jamesyu.org",
+  "image-net.org",
+  "ifdo.ca",
+  "historycooperative.org",
+  "herbsutter.com",
+  "hdsr.mitpress.mit.edu",
+  "hakaimagazine.com",
+  "gutenberg.ca",
+  "gpt3demo.com",
+  "gondwanaland.com",
+  "gaotianyu.xyz",
+  "futurism.com",
+  "extras.denverpost.com",
+  "eurekamag.com",
+  "eukaryotewritesblog.com",
+  "etienne.se",
+  "endlessvn.io",
+  "statmodeling.stat.columbia.edu",
+  "archive.nytimes.com",
+  "www.michaellight.net",
+  "www.otakustudy.com",
+  "www.baka-tsuki.org",
+  "story.californiasunday.com",
+  "thispersondoesnotexist.com",
+  "metropolitician.blogs.com",
+  "pauillac.inria.fr",
+  "asktog.com",
+  "www.unf.edu",
+];
 
 const allowSubdomainsFromGwern = [
-	'.allennlp.org',
-	'.archive.org',
-	'.archiveteam.org',
-	'.bandcamp.com',
-	'.eleuther.ai',
-	'.fandom.com',
-	'.github.io',
-	'.givewell.org',
-	'.greenspun.com',
-	'.humanprogress.org',
-	'.imagemagick.org',
-	'.mementoweb.org',
-	'.metafilter.com',
-	'.nomeata.de',
-	'.obormot.net',
-	'.tumblr.com',
-	'.xkcd.com',
-	'.wikipedia.org',
-	'.wordpress.com',
-	'.blogspot.com',
-]
+  ".allennlp.org",
+  ".archive.org",
+  ".archiveteam.org",
+  ".bandcamp.com",
+  ".eleuther.ai",
+  ".fandom.com",
+  ".github.io",
+  ".givewell.org",
+  ".greenspun.com",
+  ".humanprogress.org",
+  ".imagemagick.org",
+  ".mementoweb.org",
+  ".metafilter.com",
+  ".nomeata.de",
+  ".obormot.net",
+  ".tumblr.com",
+  ".xkcd.com",
+  ".wikipedia.org",
+  ".wordpress.com",
+  ".blogspot.com",
+];
+
+const goodNunoDomains = [
+  "github.com",
+  "nunosempere.com",
+  "alpinelinux.org",
+  "nginx.org",
+  "metaforecast.org",
+  "quantifieduncertainty.org",
+  "twitter.com",
+  "www.infer-pub.com",
```

Comment:
Some of these domains (and subdomains below) are unfortunately non-iframeable (e.g. github, twitter). So preview for them won't work. You can test that by trying to create an iframe manually elsewhere, and seeing if it'd render properly.  

One easy way to do the test would be to add them to whitelist in the following browser extension settings: https://github.com/transclude-me/extension and seing if the link previews for those sites are rendered properly

Reply:
It should be possible to create a special handler for e.g. twitter, but that would need to be a custom handler similar to Manifold one

Reply:
One could also just use a different twitter frontend, like nitter.net

Reply:
Sure, but we'd still need a custom handler to rewrite links (similar to wiki or manifold)

Reply:
actually nitter does not seem iframeable either.